tools/.vscode/launch.json

```diff
@@ -9,7 +9,7 @@
             "type": "python",
             "request": "launch",
             "stopOnEntry": true,
-            "pythonPath": "${command:python.pythonPath}",
+            "python": "${command:python.pythonPath}",
```

Comment:
I didn't make this change on purpose - seems like a vscode thing. A tooltip indicates that `pythonPath` "is not allowed" in this launch.json file but `python` is allowed.